### PR TITLE
fix(lint): various lint errors fixes

### PR DESCRIPTION
**Pull request by abhi-markan**
## Introduction
Code refactorisation

## Resolution
* Renamed `err` to `error` for code consistency.

## Miscellaneous
* Lint fixes
* Dependencies updates

### CODE REVIEW

**Review by anmol111pal — APPROVED** [review]

File: utils/data-migration/export-tables.js [hunk 1]
```diff
@@ -100,8 +100,9 @@ const exportAllCollections = async (collections) => {
 
       obj.files.push(table);
       // write the file to the ./exports directory
-      fs.writeFile(`${exportDirectory}/${date}/${table}.json`, JSON.stringify(result, null, 2), (error) => {
-        if (error) throw error;
+      // Defined `e` since `error` is defined on a higher scope
+      fs.writeFile(`${exportDirectory}/${date}/${table}.json`, JSON.stringify(result, null, 2), (e) => {
```
Comment (ChristianMcc) [inline utils/data-migration/export-tables.js:104]: probably more readable to have it defined as writeFileError rather than e

Comment (abhi-markan) [reply utils/data-migration/export-tables.js:104]: Have renamed.

Comment (abhi-markan) [reply utils/data-migration/export-tables.js]: Have updated. Thanks

**Review by ChristianMcc — APPROVED** [review]

File: gef-ui/server/index.js [hunk 2]
```diff
@@ -105,13 +105,13 @@ app.use('/', routes);
 app.use('/assets', express.static(path.join(__dirname, '..', 'public')));
 
 // eslint-disable-next-line no-unused-vars
```
Comment (Zainzzkk) [inline gef-ui/server/index.js:107]: do we need this here?

File: gef-ui/server/models/application.js [hunk 3]
```diff
@@ -127,10 +127,10 @@ class Application {
       }
 
       return application;
-    } catch (err) {
+    } catch (error) {
       // eslint-disable-next-line no-console
```
Comment (Zainzzkk) [inline gef-ui/server/models/application.js:131]: do we need this since it's console.error

File: gef-ui/server/index.js (hunk 2 above)
Comment (abhi-markan) [reply gef-ui/server/index.js:107]: All of these will be removed as part of separate PR.

File: gef-ui/server/models/application.js (hunk 3 above)
Comment (abhi-markan) [reply gef-ui/server/models/application.js:131]: Same as above, but no we don't.

**Pull request merged**